### PR TITLE
fixed issues discussed in the last meeting and fixed encoding errors.

### PR DESCRIPTION
**Pull request by dITySoftware**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,9 +5,11 @@ dist: bionic
 
 before_install:
   - sudo apt-get update -y
-  - sudo apt-get install -y exim4 procps cpulimit ansible mailutils python3-pip python3-kafka
+  - sudo apt-get install -y exim4 procps cpulimit ansible mailutils python3-pip
   - git clone https://github.com/ait-aecid/aminer-ansible.git roles/aminer
+  - cp $PWD/source/root/etc/aminer/conf-available/generic/ApacheAccessModel.py $PWD/source/root/usr/lib/logdata-anomaly-miner/ApacheAccessModel.py
```
Comment (landauermax) [inline .travis.yml:10]: Why is this necessary? This destination directory is not meant to contain parsing models.

Comment (dITySoftware) [reply .travis.yml:10]: This is necessary, because the template_config.py imports the ApacheAccessModel and the stated config is tested with log lines from the AMiner Demo. Otherwise this model can not be imported in the config.

What is the appropriate way of importing these generic models?
